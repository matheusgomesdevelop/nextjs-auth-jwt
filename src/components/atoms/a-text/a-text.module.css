.a_text {
    font-size: var(--fs-text-md);
    color: var(--gray-500);
    font-weight: var(--fw-regular);
}

.a_text__error {
    color: var(--red-500);
    font-size: var(--fs-text-sm);
}