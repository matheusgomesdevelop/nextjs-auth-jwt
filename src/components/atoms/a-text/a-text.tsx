import { HTMLAttributes } from "react";

interface I_AText extends HTMLAttributes<HTMLParagraphElement> {}

export const A_Text = ({ children }: I_AText) => {
  return <p data-testid="a-text">{children}</p>;
};
