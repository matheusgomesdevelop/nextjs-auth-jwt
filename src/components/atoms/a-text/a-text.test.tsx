import renderer from "react-test-renderer";

// Atoms
import { A_Text } from "./a-text";

describe("Deve renderizar o a-text, corretamente", () => {
  it("Deve preservar a estrutura visual do a-text", () => {
    const a_text_label = "";
    const get_a_text = renderer
      .create(<A_Text>{a_text_label}</A_Text>)
      .toJSON();

    expect(get_a_text).toMatchSnapshot();
  });
});
