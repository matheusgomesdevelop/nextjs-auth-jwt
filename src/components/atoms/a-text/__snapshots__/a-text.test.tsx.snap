// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar o a-text, corretamente Deve preservar a estrutura visual do a-text 1`] = `
<p
  data-testid="a-text"
/>
`;
